index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Portfolio</title>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+	<div class="app-container">
+		<nav class="app-nav">
+			<a id="logo" href="#">My logo</a>
+			<i id="menu-icon" class="bi bi-list"></i>
+		</nav>
+		<main class="app-greeting">
+			<h1>I'm Sane great to see you!</h1>
+			<p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work
+				and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+			<p id="lets-connect">LET'S CONNECT</p>
+			<div class="socials">
+				<a href="#"><i class="bi bi-twitter"></i></a>
+				<a href="#"><i class="bi bi-linkedin"></i></a>
+				<a href="#"></a><i class="bi bi-github"></i></a>
+			</div>
```

Comment:
- [ ] `required`
- [ ] You missed one of the `social icons` please add the missing icons so that you match the figma template 
- [ ] see screenshots below

figma template
![image](https://user-images.githubusercontent.com/66230306/192023436-4a5ce291-6181-4b81-ba0b-47afc0baf177.png)

your template
![missing iccones](https://user-images.githubusercontent.com/66230306/192023661-17c74ef5-213b-463b-9f8e-72c108822f40.png)

- [ ] `note` when implimenting please make sure there in the same order as the template